flight/Modules/VtolPathFollower/vtol_follower_control.c

```diff
@@ -627,6 +517,148 @@ int32_t vtol_follower_control_attitude(float dT)
 	return 0;
 }
 
+
+static float loiter_deadband(float input) {
+	const float CMD_THRESHOLD = 0.2f;
```

Comment:
I think this is a bit much, i.e., quite a bit of stick deflection is needed for the UAV to start moving. Given that the sticks are spring centered, it should IMO just be big enough to make sure that an imperfect calibration doesn't cause drift. How about 0.1 or even 0.05?